File: engine/engine-model/src/main/scala/net/server/channel/packet/guild/JoinGuildPacket.scala
package net.server.channel.packet.guild

class JoinGuildPacket(private var _type: Byte, private var _guildId: Int, private var _playerId: Int) extends BaseGuildOperationPacket(_type = _type) {
  def guildId: Int = _guildId

  def playerId: Int = _playerId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/guild/JoinGuildPacket.java
package net.server.channel.packet.guild;

public class JoinGuildPacket extends BaseGuildOperationPacket {
   private final Integer guildId;

   private final Integer playerId;

   public JoinGuildPacket(Byte theType, Integer guildId, Integer playerId) {
      super(theType);
      this.guildId = guildId;
      this.playerId = playerId;
   }

   public Integer guildId() {
      return guildId;
   }

   public Integer playerId() {
      return playerId;
   }
}
